desktop/ui/src/main/java/org/datacleaner/widgets/properties/MultipleInputColumnsPropertyWidget.java

```diff
@@ -228,8 +228,8 @@ private void updateComponents() {
 
     private void updateComponents(final InputColumn<?>[] value) {
         // fetch available input columns
-        final List<InputColumn<?>> availableColumns = getAnalysisJobBuilder().getAvailableInputColumns(
-                getComponentBuilder(), _dataType);
+        final List<InputColumn<?>> availableColumns = getAnalysisJobBuilder()
+                .getAvailableInputColumns(getComponentBuilder(), _dataType);
```

Comment:
This isn't really weird, but it seems pretty aggressive. The old line was 96, and I believe we use 120 as the max width.